src/Handler/Setup.php

```diff
@@ -27,13 +30,8 @@ class Setup
     private $dependencies;
     private $options;
 
-    public function __construct(
-        string $cwd,
-        AppName $appName,
-        EnvEnum $envName,
-        \Traversable $dependencies,
-        array $options = []
-    ) {
+    public function __construct(string $cwd, AppName $appName, EnvEnum $envName, \Traversable $dependencies, array $options = [])
```

Comment:
Here you can see both PHPStorm (through method line indicators) and code sniffer detected the line is too long.

<img width="950" alt="capture d ecran 2016-10-31 a 11 47 39" src="https://cloud.githubusercontent.com/assets/2211145/19851779/ec568cee-9f5f-11e6-80c8-6732684d1929.png">